homeassistant/components/tuya/cover.py

```diff
@@ -193,11 +200,11 @@ def __init__(
             self._attr_supported_features |= SUPPORT_OPEN | SUPPORT_CLOSE
         elif device.function[description.key].type == "Enum":
             data_type = EnumTypeData.from_json(device.function[description.key].values)
-            if "open" in data_type.range:
+            if "open" in data_type.range or "FZ" in data_type.range:
```

Comment:
I think that adding if/else constructs everywhere to adjust the command for a specific device is not the way to go.

It would become a mess really quick when allowing/adding multiple of these exceptions.

So, what we should do instead is: see if Tuya can standardize the device (ask them in a support ticket), or, resolve this in a more configurable way (eg using/extending the entity descriptions).

Reply:
Changed to a configuration approach.